robots/barr-e/ros2/src/barr_e_mapping/src/robotick_bridge_node.cpp

```
@@ -1,7 +1,65 @@
+	RobotickBridgeNode();
+	~RobotickBridgeNode();
+
+	void on_robotick_bridge_tick(MappingRosBridgeOutputs& outputs, RobotickBridgeState& state, const robotick::TickInfo& tick_info);
+	void run_engine_thread();
+
+  private:
+	void on_odom(const nav_msgs::msg::Odometry::SharedPtr msg);
+	void on_map(const nav_msgs::msg::OccupancyGrid::SharedPtr msg);
+	void on_tf(const tf2_msgs::msg::TFMessage::SharedPtr msg);
+
+  private:
```

Clang (14.0.6)

[warning] redundant access specifier has the same accessibility as the previous access specifier

(readability-redundant-access-specifiers)